modules/authentication/client-react/access/jwt/components/Loading.jsx

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import { LayoutCenter } from '@module/look-client-react';
 
-export default (
+const Loading = () => (
   <LayoutCenter>
     <div className="text-center">App is loading...</div>
```

Comment:
Can we move this text to a file with localization?

Reply:
done